hail_scripts/reference_data/combine.py

```diff
@@ -95,13 +103,32 @@ def update_joined_ht_globals(
 
 def join_hts(datasets, version, reference_genome='37'):
     # Get a list of hail tables and combine into an outer join.
-    hts = [get_ht(dataset, reference_genome) for dataset in datasets]
+    hts = [
+        get_ht(dataset, reference_genome)
+        for dataset in datasets
+        if 'coverage' not in dataset
+    ]
     joined_ht = functools.reduce(
-        (lambda joined_ht, ht: joined_ht.join(ht, 'outer')), hts
+        (lambda joined_ht, ht: joined_ht.join(ht, 'outer')),
+        hts,
     )
 
+    # NB: coverage datasets are keyed by locus rather than locus
+    # and alleles, so we cannot join.  Instead we annotate w/ locus as
+    # as the key.
+    coverage_hts = [
+        (dataset, get_ht(dataset, reference_genome))
+        for dataset in datasets
+        if 'coverage' in dataset
+    ]
+    for dataset, coverage_ht in coverage_hts:
+        joined_ht.annotate(dataset=coverage_ht[coverage_ht.locus][dataset])
+
```

Comment:
This makes sense. I was wondering if we could compare the key values, when they match --  join, when one is a subset of the other annotate, but that seems like over-engineering for something that runs once.

Reply:
yeah.. I actually thought that approach might be preferable as well.  Do you think that we'll have other datasets down the line that aren't coverage but are keyed only by `locus`?

Reply:
Nothing immediately comes to mind. The interval ref dataset is basically locus but just a range, `1-10` instead of `1`, `2`, `3`, `4`,... I imagine most datasets would be keyed by ranges and added to the interval dataset.